presto-main/src/test/java/com/facebook/presto/execution/buffer/TestArbitraryOutputBuffer.java

```diff
@@ -937,10 +937,13 @@ public void testForceFreeMemory()
         assertTrue(memoryManager.getBufferedBytes() > 0);
         buffer.forceFreeMemory();
         assertEquals(memoryManager.getBufferedBytes(), 0);
-        // adding another page after buffer.forceFreeMemory()
-        // should have no effect in terms of memory usage
-        addPage(buffer, createPage(1));
-        assertEquals(memoryManager.getBufferedBytes(), 0);
+        try {
+            addPage(buffer, createPage(1));
+            fail("Adding a page to a closed memoryManager should have failed");
+        }
+        catch (IllegalStateException e) {
+            // expected
+        }
```

Comment:
validate `assertEquals(memoryManager.getBufferedBytes(), 0);` incase add pages bumped memory and then threw